build.gradle

```diff
@@ -23,24 +23,22 @@ ext {
     earliestSupportedMajor: '2020.1',
     // Most recent minor versions of all major releases
     // between earliest supported (incl.) and latest stable (excl.), used for binary compatibility checks
-    latestMinorsOfOldSupportedMajors: ['2020.1.4'],
-    latestStable: '2020.2.3',
+    latestMinorsOfOldSupportedMajors: ['2020.1.4', '2020.2.4'],
```

Comment:
From now on, I hope only just changes in this section (IntelliJ version) will be necessary when bumping versions... plus ofc removing deprecated stuff etc.

Reply:
But no other changes to build.gradle or CI config should be necessary (at least in principle)